lib/jetstream/api/object.ex

```diff
@@ -16,9 +16,7 @@ defmodule Jetstream.API.Object do
         name: stream_name(bucket_name),
         subjects: stream_subjects(bucket_name),
         description: Keyword.get(params, :description),
-        max_msgs_per_subject: Keyword.get(params, :history, 1),
```

Comment:
After writing my test for storing and retrieving a large file, I couldn't get the test to pass. I was troubleshooting with the `nats` CLI tool and realized that when I used the CLI to add a file to my test bucket it also couldn't download the file. Eventually I realized it's because I was creating the bucket with the wrong parameters.